### DISCUSSION TITLE
osu!lazer is getting cursed over time

### DISCUSSION THREAD

**Discussion by fantasyzhjk** [ppy/osu#21919] [Q&A]
### Type

Other

### Bug description

osu!lazer is getting cursed over time until the game goes to blackscreen

### Screenshots or videos

https://youtu.be/PZMoEWrNnXo

### Version

osu 2022.1227.0.0 on .NET 6.0.2

### Logs

[input.log](https://github.com/ppy/osu/files/10317291/input.log)
[network.log](https://github.com/ppy/osu/files/10317292/network.log)
[performance.log](https://github.com/ppy/osu/files/10317293/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10317294/runtime.log)
[updater.log](https://github.com/ppy/osu/files/10317295/updater.log)
[database.log](https://github.com/ppy/osu/files/10317296/database.log)

**Comment by bdach** [comment]
Please try updating your graphics drivers and seeing if it helps.

**Reply by frenzibyte** [reply]
Marking as answered until a response is given.

**Reply by bdach** [reply]
@frenzibyte don't think it's how discussions are supposed to work. let's mark resolved when _the user_ says it's resolved.

**Reply by frenzibyte** [reply]
Been following a personal policy of ruling out any threads which suffer from a seemingly known cause, to separate it from actual unanswered discussions. If the author claims it's not the case, we can just unmark the discussion.

**Reply by fantasyzhjk** [reply]
Even after updating the graphics card driver, this issue remains unfixed. It is also strange that other games do not have the same problem. This issue seems to be specific to osu!lazer.

**Reply by fantasyzhjk** [reply]
The game always runs for a while and then suddenly goes black, and OBS can no longer capture the game screen.
